package.json

```diff
@@ -21,42 +21,42 @@
     "react": "^0.14.0 || ^15.0.0"
   },
   "devDependencies": {
-    "babel-cli": "^6.14.0",
-    "babel-core": "^6.14.0",
-    "babel-eslint": "^6.1.2",
-    "babel-loader": "^6.2.5",
-    "babel-preset-es2015": "^6.14.0",
-    "babel-preset-react": "^6.5.0",
-    "babel-preset-stage-0": "^6.5.0",
-    "babel-register": "^6.14.0",
+    "babel-cli": "^6.22.2",
+    "babel-core": "^6.22.1",
+    "babel-eslint": "^7.1.1",
+    "babel-loader": "^6.2.9",
+    "babel-preset-es2015": "^6.22.0",
+    "babel-preset-react": "^6.22.0",
+    "babel-preset-stage-0": "^6.22.0",
+    "babel-register": "^6.22.0",
     "expect": "^1.14.0",
-    "gh-pages": "^0.11.0",
+    "gh-pages": "^0.12.0",
     "gx": "^1.0.0-rc3",
-    "jsdom": "^9.5.0",
+    "jsdom": "^9.10.0",
     "json-loader": "^0.5.4",
-    "jsonp": "^0.2.0",
-    "mocha": "^3.0.2",
+    "jsonp": "^0.2.1",
+    "mocha": "^3.2.0",
     "mocha-jsdom": "^1.1.0",
     "object-assign": "^4.1.0",
-    "onchange": "^3.0.2",
-    "react": "^15.3.1",
-    "react-addons-test-utils": "^15.3.1",
+    "onchange": "^3.2.1",
+    "react": "^15.4.2",
+    "react-addons-test-utils": "^15.4.2",
     "react-component-permutations": "^1.0.0-beta2",
-    "react-docgen": "^2.10.0",
-    "react-dom": "^15.3.1",
-    "react-element-to-jsx-string": "^4.1.0",
+    "react-docgen": "^2.13.0",
+    "react-dom": "^15.4.2",
+    "react-element-to-jsx-string": "^6.0.0",
     "react-geomicons": "^2.0.5",
-    "react-router": "^2.8.0",
+    "react-router": "^3.0.2",
     "reflexbox": "^2.1.0",
-    "standard": "^8.0.0",
+    "standard": "^8.6.0",
     "static-react": "^3.1.0",
-    "webpack": "^1.13.2",
-    "webpack-dev-server": "^1.15.1"
+    "webpack": "^1.14.0",
```

Comment:
I think that's better left for a separate PR. My goal was to get all the dependencies upgraded within their defined semver range and make as little code changes as possible in doing so